code/modules/hydroponics/plants_alien.dm

```diff
@@ -141,7 +141,7 @@
 
 /datum/plant/artifact/dripper
 	name = "Dripper"
-	special_icon = "dripper"
+	override_icon_state = "Dripper"
```

Comment:
Possible to get the casing here standardized? Note that `/datum/plant/artifact/peeker` has `override_icon_state` as `"peeker"` (i.e. lowercase), whereas this one is `"Dripper"` (i.e. capitalized).

Reply:
Looking into further, is there a reason this is needed at all? By which I mean: isn't this the `icon_state` that is going to be used anyway? I understand the need for those with e.g. spaces in their names, but most don't have that.

Reply:
I'll make these changes now :)

Reply:
Resolved in latest commit.